src/store/preferences/index.ts

```diff
@@ -10,8 +11,25 @@ export default {
     swimlaneStates: jsonstore.get("swimlaneStates", {}),
     search: "",
     taskEditorSectionIdAndTask: null,
+    dateFormatString: "yyyy-MM-dd"
   } as State,
-  getters: {},
+  getters: {
+    isSaveDisabled: (state: State) => (date: string) => {
```

Comment:
Is this for the currently selected task? This has a bit of a code smell to it, but need to know it's purpose a bit more